src/widgets/search-box/__tests__/search-box-test.js

```diff
@@ -24,10 +24,17 @@ describe('searchBox()', () => {
       query: '',
     };
     helper = {
+      setState(nextState) {
```

Comment:
🤦‍♂️ We should not reimplement the helper in the tests. Let's take care of that in the next iteration :)